gcalcli/utils.py

```diff
@@ -143,3 +143,12 @@ def agenda_time_fmt(dt, military):
     hour_min_fmt = '%H:%M' if military else '%I:%M'
     ampm = '' if military else dt.strftime('%p').lower()
     return dt.strftime(hour_min_fmt).lstrip('0') + ampm
+
+
+def is_all_day(event):
+    # XXX: currently gcalcli represents all-day events as those that both begin
```

Comment:
Good point!  Is there a field in the API event structure which disambiguates these cases?